File: Hackerrank/Problem Solving/AngryProf.py
def angryProf(tc):
    for i in range(tc):
        s=0
        n,k=map(int,input().split())
        time=list(map(int,input().split()))
        for j in range(n):
            if(time[j]<=0):
                s+=1
        if(s>=k):
            print("NO")
        else:
            print("YES")


try:
    tc=int(input())
except:
    print("ERROR! Enter numeric input only!")
    quit()
angryProf(tc)

File: Hackerrank/Problem Solving/AngryProf.cpp
#include<iostream>
using namespace std;
int x,i,j,p,y;


int main()
{
    int t;
    cin>>t;
    int n[t],k[t],a[t][1000];
    
    for(i=0;i<t;i++)
    {
        cin>>n[i]>>k[i];
        int x=n[i];
        for(j=0;j<x;j++)
            cin>>a[i][j];
        
    }
       
        for(i=0;i<t;i++)
        { 
            p=0;  
            y=n[i];
            for(j=0;j<y;j++)
            {
                if(a[i][j]<=0)
                    p++;
            }
            if(p>=k[i])
                cout<<"NO"<<endl;
            else
                cout<<"YES"<<endl;
        }
    }
    
    

